sdk/core/core-http/test/urlTests.ts

```diff
@@ -97,7 +97,15 @@ describe("URLQuery", () => {
     });
 
     it(`with "A=="`, () => {
-      assert.strictEqual(URLQuery.parse("A==").toString(), "");
+      assert.strictEqual(URLQuery.parse("A==").toString(), "A==");
```

Comment:
do we have to worry about doing a test beyond round-tripping? I guess if it round trips, then we're pretty confident that the parsed form is correct.

Reply:
yeah I think round-tripping is enough.